src/main/java/seedu/address/flashcard/Answer.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.flashcard;
+
+/**
+ * Represents an Answer in a flashcard.
+ */
+public class Answer {
+    final private String answer;
+
+    public Answer(String answer) {
+        this.answer = answer;
+    }
+
+    /**
+     * Checks the given userAnswer with the correct answer.
+     * This is done by strictly comparing lower case string equality.
+     * @param userAnswer the user's answer.
+     * @return true if the user's answer is equal to the actual answer.
+     */
+    public boolean checkAnswer(String userAnswer) {
+        String lowerCaseActualAnswer = answer.toLowerCase();
+        String lowerCaseUserAnswer = userAnswer.toLowerCase();
+        return lowerCaseActualAnswer.equals(lowerCaseUserAnswer);
+    }
+
+    /**
+     * Gets the correct answer.
+     * @return the correct answer.
+     */
+    public String getAnswer() {
+        return answer;
+    }
```

Comment:
Would be good if you can include the `toString()` methods for the Answer and Question class, that way when it is implemented in the GUI it would be easier to render it seperately (since the current way to access the string form of the question and answer is through the Flashcard class). 

The current `toString()` method in FlashCard is still useful when interacting with the application through CLI 👍

Reply:
Yeah,  I think can return answer instead of String.